### ISSUE TITLE
ysaFunc() not working- Standard deviation too high for beta distribution

### ISSUE THREAD

**Issue by Ellie7** [Ellie7/parrot-proj#23]
tried amending dataframe with `sqrt((1 - pi)*pi) - 0.001` but don't think I've done it right / tried a variety of things and can't get it working :/ 
its line 55/56 in the YSA demography script

**Comment by tdjames1** [comment]
It's a bit messy but line 55 becomes this:
```
piSD <- c((total_summary$se_hatch[1]), (total_summary$se_nestling_surv[1]), 0.2, sqrt((1 - pi[4])*pi[4]) - 0.001, sqrt((1 - pi[5])*pi[5]) - 0.001) # 
```

**Comment by tdjames1** [comment]
That's not totally satisfactory but I think it'll do. You can put in a comment that the SD generated from the imputation (0.36363) was too high to generate beta values so you replaced it with a value slightly less than the maximum allowed value (0.3555892).

**Comment by Ellie7** [comment]
Thank you so much will do. 
Fixed with commit 1d938bc
